.travis.yml

```diff
@@ -63,4 +64,6 @@ script:
   - make
   - make check
   - make deb
+  - docker build -f docker/Dockerfile -t centos:7 .
+  - docker run --rm -v $PWD:/home centos:7 make docker-rpm
```

Comment:
Yeah I reviewed this one later. IMO the approach you have here is complicated by the fact you are not using build matrix. I would strongly recommend to go with the build matrix, then update this P.R. based on it.